ct/bookstack.sh

```diff
@@ -60,30 +60,45 @@ check_container_resources
 if [[ ! -d /opt/bookstack ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
 RELEASE=$(curl -s https://api.github.com/repos/BookStackApp/BookStack/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
 if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+
   msg_info "Stopping Apache2"
-  systemctl stop apache2
+  systemctl stop apache2t 
   msg_ok "Services Stopped"
 
   msg_info "Updating ${APP} to ${RELEASE}"
-  cp /opt/bookstack/.env /opt/.env
+  cp -r /opt/bookstack/ /opt/bookstack-backup 
+  tar -czf /opt/bookstack_backup_${RELEASE}.tar.gz /opt/bookstack/.env /opt/bookstack/public/uploads /opt/bookstack/storage/uploads /opt/bookstack/themes &>/dev/null
+  mysqldump -u root bookstack > /opt/bookstack_backup_${RELEASE}.sql
```

Comment:
here bookstack_backup, above bookstack-backup?

Reply:
changed tar and sql to `bookstack-backup`